network-mux/src/Network/Mux/Types.hs

```diff
@@ -158,6 +170,13 @@ newtype MiniProtocolDispatch ptcl m =
         MiniProtocolDispatch (Array (MiniProtocolId ptcl, MiniProtocolMode)
                                     (StrictTVar m BL.ByteString))
 
+newtype MiniProtocolInitiatorControlTable ptcl m a =
+        MiniProtocolInitiatorControlTable (Array (MiniProtocolId ptcl)
+                                                 (StrictTMVar m (), StrictTMVar m a))
```

Comment:
add haddocks to document what both `StrictTMVar` are for

Reply:
Will address.